scale/ingest/strike/monitors/dir_monitor.py

```diff
@@ -269,7 +269,8 @@ def _process_file(self, file_name, ingest):
                 logger.info('Transfer complete: %s', file_path)
 
         if ingest.status == 'TRANSFERRED':
-            ingest_path = os.path.join(self._ingest_dir, file_name)
+            # TODO: do the same for S3?
```

Comment:
should probably verify one way or another

Reply:
Couldn't find any evidence S3 would have an issue. Removed the TODO.